src/components/ShowCard.tsx

```diff
@@ -0,0 +1,26 @@
+import { IShow } from "../types";
+import { pTagRemoval } from "../utils/pTagRemoval";
+
+export function ShowCard(props: IShow): JSX.Element {
+  return (
+    <div className="showCard" key={props.id}>
+      <h2>{props.name}</h2>
+      <div>
+        {props.image !== null && (
+          <img src={props.image.medium} alt="snapshot of a show" />
```

Comment:
Good having an alt description. Could say the show name it is referring to, to make it more useful as an alt text